airqo_monitor/models.py

```diff
@@ -1,5 +1,67 @@
 from django.db import models
 
-# Create your models here.
+
 class Greeting(models.Model):
```

Comment:
let's trash this (and the corresponding code in views.py)

Reply:
will remove in another pr